PowerEditor/src/menuCmdID.h

```diff
@@ -175,6 +175,7 @@
     #define    IDM_EDIT_AUTOCOMPLETE_CURRENTFILE                (50000 + 1)
     #define    IDM_EDIT_FUNCCALLTIP                             (50000 + 2)
     #define    IDM_EDIT_AUTOCOMPLETE_PATH                       (50000 + 6)
+    #define    IDM_EDIT_FUNCCALLTIP_NEXT                        (50000 + 10)

```

Comment:
Wouldn't be 50000 + 3 more logical? The gap before IDM_EDIT_AUTOCOMPLETE_PATH was left intentional I guess.

Reply:
50003 is already in use as is up to 50005.  Up to 50009 are in, but commented.  I wanted to avoid any issues:

https://github.com/notepad-plus-plus/notepad-plus-plus/blob/f2c4ecd7e1bfd5d7d5c14c854438c02f2ba0909a/PowerEditor/src/resource.h#L425-L435

Cheers.